code/rd-vanilla/tr_surface.cpp

```diff
@@ -1111,7 +1111,7 @@ static void RB_SurfaceBeam( void )
 	}
 
 	qglBegin( GL_TRIANGLE_STRIP );
-	for ( i = 0; i <= NUM_BEAM_SEGS; i++ ) {
+	for ( i = 0; i < NUM_BEAM_SEGS; i++ ) {
```

Comment:
Didn't test anything, but `<=` looks intentional to close it off, why else would there be a modulo? Did you test these changes?

Reply:
> Didn't test anything, but `<=` looks intentional to close it off, why else would there be a modulo? Did you test these changes?

^ This is correct. Its used intentionally for sealing.

Reply:
Thanks for confirming, that makes sense. I was having trouble visualising what vertices were being emitted, and it's hard to test ingame since nothing uses them and it doesn't seem to render anything anyway :shrug: